nipype/interfaces/ants/registration.py

```diff
@@ -1034,3 +1034,114 @@ def _list_outputs(self):
         if len(self.inputs.save_state):
             outputs['save_state'] = os.path.abspath(self.inputs.save_state)
         return outputs
+
+
+class MeasureImageSimilarityInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(
+         2, 3, 4,
+         argstr='--dimensionality %d', position=1,
+         desc='Dimensionality of the fixed/moving image pair',
+         )
+    fixed_image = File(
+         exists=True, mandatory=True,
+         desc='image to which the moving image is warped',
+         )
+    moving_image = File(
+         exists=True,mandatory=True,
+         desc='image to apply transformation to (generally a coregistered functional)',
+         )
+    metric = traits.Enum(
+         "CC", "MI", "Mattes", "MeanSquares", "Demons", "GC",
+         argstr="%s", mandatory=True,
+         )
+    metric_weight = traits.Float(requires=['metric'], desc='',mandatory=True,)
```

Comment:
Space after commas, no comma before close paren. Any chance of a non-empty `desc`?